notebooks/community/managed_notebooks/sentiment_analysis/Sentiment_Analysis.ipynb

```diff
@@ -0,0 +1,1044 @@
+{
```

Comment:
<p>Use "Vertex AI's managed instances" rather than "Vertex-AIs managed instances"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>resolved</p>